livekit-ffi/protocol/ffi.proto

```diff
@@ -96,9 +98,14 @@ message FfiRequest {
     NewSoxResamplerRequest new_sox_resampler = 33;
     PushSoxResamplerRequest push_sox_resampler = 34;
     FlushSoxResamplerRequest flush_sox_resampler = 35;
-
     SendChatMessageRequest send_chat_message = 36;
     EditChatMessageRequest edit_chat_message = 37;
+
+    // RPC
+    PerformRpcRequest perform_rpc = 38;
```

Comment:
what do you think about naming this method send_rpc ?

Reply:
I went back and forth on this but I think `perform` is clearer because it's a method invocation that will wait for a response. `send` feels more like a one-way method. Why do you think it should be called `send`?